src/platform/qpg6100/qpg6100Config.cpp

```diff
@@ -284,6 +246,47 @@ CHIP_ERROR QPG6100Config::ForEachRecord(Key firstKey, Key lastKey, bool addNewRe
     return err;
 }
 
+CHIP_ERROR QPG6100Config::MapNVMError(qvStatus_t aStatus)
+{
+    CHIP_ERROR err;
+
+    if (aStatus == QV_STATUS_NO_ERROR)
+    {
+        ChipLogProgress(NotSpecified, "No ERROR");
+    }
+    else
+    {
+        ChipLogProgress(NotSpecified, "ERROR %d", (uint16_t) aStatus);
+    }
+    switch (aStatus)
+    {
+    case QV_STATUS_NO_ERROR: {
```

Comment:
nit: I believe extra `{}` are not needed and removing them could decrease vertical code size.